### PR TITLE
Add command to run CGUs only on modified services

### PR DESCRIPTION
**Pull request by LucasVerneyDGE**
Hi,

When adding new services, it is convenient to be able to run in a single command the checks on the modified services. Here is a proposal for such a npm command. Let me know what you think about it and whether it might be of interest :)

Best,

### CODE REVIEW

**Comment by LucasVerneyDGE** [comment]
Updated with this logic.

File: src/index.js [hunk 1]
```diff
@@ -23,6 +32,9 @@ let serviceIds = args.filter(arg => !arg.startsWith('--'));
     return isServiceDeclared;
   });
 
+  if (modifiedOnly && !serviceIds.length) {
+    return;
+  }
```
Comment (Ndpnt) [inline src/index.js:37]: Can you add a new line after the `}`?

File: src/index.js [hunk 2]
```diff
@@ -23,6 +32,9 @@ let serviceIds = args.filter(arg => !arg.startsWith('--'));
     return isServiceDeclared;
   });
 
+  if (modifiedOnly && !serviceIds.length) {
+    return;
```
Comment (Ndpnt) [inline src/index.js:36]: Add a log, something like `logger.warn('No services have been modified');`

File: src/tools/index.js [hunk 3]
```diff
@@ -0,0 +1,19 @@
+import path from 'path';
```
Comment (Ndpnt) [inline src/tools/index.js:1]: Rename `tools` in `utils`, it seems more idiomatic to me.

File: src/tools/index.js [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+import simpleGit from 'simple-git';
+
+export async function getModifiedServices(rootPath) {
```
Comment (Ndpnt) [inline src/tools/index.js:4]: As the path for `services` is always the same no matter where we call this function, I'm in favor of not passing `rootPath` as parameter but defining it directly inside this module.

File: src/index.js [hunk 5]
```diff
@@ -1,19 +1,28 @@
...
 const schedule = args.includes('--schedule');
-let serviceIds = args.filter(arg => !arg.startsWith('--'));
 
 (async () => {
   const app = new CGUs();
   app.attach(logger);
   await app.init();
 
+  let serviceIds = args.filter(arg => !arg.startsWith('--'));
+  if (modifiedOnly) {
```
Comment (Ndpnt) [inline src/index.js:20]: Add a newline before the `if` to create meaningful blocs

**Review by Ndpnt — CHANGES_REQUESTED** [review]
Hi @LucasVerneyDGE,
Thanks for this CLI improvement!
Just some minors changes and it's GTM for me.

**Comment by Ndpnt** [comment]
Also, can you please rebase master branch.

**Comment by LucasVerneyDGE** [comment]
Thanks for the review, PR should be updated!

**Review by MattiSG — APPROVED** [review]
Not tested locally, but discussed & read. Thanks @LucasVerneyDGE!

As complexity in handling services increases, for example when implementing #156, it will be appropriate to move the `getModifiedServices` logic in a `services` or `documents` module,  rather than `utils` 🙂

**Pull request merged**